retriever/bin/retriever.js

```diff
@@ -47,15 +57,34 @@ export default {
 
     // Timestamp to measure file retrieval performance (from cache and from SP)
     const fetchStartedAt = performance.now()
+    const { ownerAddress, error: ownerLookupError } =
+      /** @type {{ ownerAddress: string; error: string }} */ await getOwnerByRootCid(
+        env,
+        rootCid,
+      )
+    if (ownerLookupError) {
+      console.error(ownerLookupError)
+      return new Response(ownerLookupError, { status: 404 })
+    }
 
+    if (
+      !ownerAddress ||
+      !Object.prototype.hasOwnProperty.call(OWNER_TO_URL, ownerAddress)
+    ) {
+      return new Response(
+        `No PDP URL configured for owner address: ${ownerAddress}`,
```

Comment:
I think this message could be confusing to people retrieving the content. I propose to rephrase it as follows:

```suggestion
        `Unsupported Storage Provider (PDP ProofSet Owner): ${ownerAddress}`,
```